webapp/src/components/gallery/gallery.tsx

```diff
@@ -90,6 +99,10 @@ const Gallery = (props: Props): JSX.Element => {
                     />
                 </div>
             }
+            {limited.length > 0 && <div className='limitedCard'>
+                <div className='limitedCard-title'>{intl.formatMessage({id: 'limitedCard.title', defaultMessage: 'Cards Hidden'})}</div>
+                <Button title='limited-card-count'>{limited.length}</Button>
+            </div>}
```

Comment:
Move to a component, then just call component where desired. Encapsulates the limit card message feature